src/main/java/org/assertj/core/error/ShouldContainCharSequence.java

```diff
@@ -39,6 +42,36 @@ public static ErrorMessageFactory shouldContain(CharSequence actual, CharSequenc
                                    StandardComparisonStrategy.instance());
   }
 
+  public static ErrorMessageFactory shouldContain(Throwable actual, CharSequence sequence) {
+    String format = "%n" +
+                    "Expecting:%n" +
```

Comment:
I would reword this to ` "Expecting throwable message:%n" ` as we know the following string is not any string but a throwable message